e2e-tests/specs/all.e2e.js

```diff
@@ -368,8 +368,8 @@ describe('Filter and Search Behaviors', async () => {
         await clearAll.click();
         const cardRes = await $('.consonant-LoadMore-text').getText();
         expect(cardRes).toEqual('Showing 5 of 8 cards');
-        // const newUrl = await browser.getUrl();
-        // expect(newUrl).toEqual(`${serverPath}/html/e2e/e2e-grid.html?state=${state}`);
+        const newUrl = await browser.getUrl();
+        expect(newUrl.replaceAll('%3D', '=')).toEqual(`${serverPath}/html/e2e/e2e-grid.html?state=${state}`);
```

Comment:
Instead of using a replace() function, why not just change the expected string:
`expect(newUrl).toEqual(`${serverPath}/html/e2e/e2e-grid.html?state%3D${state}`);`

Reply:
because the equals signs I'm replacing that are causing me issues are two ones that get postpended into the state when it is parsed specifically by the github production version of the e2e tests - there are no computed %3Ds in the local tests so changing the expected string breaks local tests